CharacterMap/CharacterMap/Core/FontVariant.cs

```diff
@@ -114,7 +113,7 @@ public IReadOnlyList<Character> GetCharacters()
                     {
                         characters.Add(new Character
                         {
-                            Char = char.ConvertFromUtf32((int)i),
+                            Char = new string((char)i, 1),
```

Comment:
@JohnnyWestlake 

This seems to cause "Segoe UI Emoji" not rendering a part of its characters. 

![image](https://user-images.githubusercontent.com/3304703/74634111-33905100-519e-11ea-9df3-09ba03a97875.png)

Reply:
I work around it in https://github.com/EdiWang/Character-Map-UWP/commit/6ae184950704416cc44c291a784af0af1bbd8fdc 

But the code is not nice and I am not sure if this will have problems. @JohnnyWestlake please review, thanks!